pkl-core/src/test/files/LanguageSnippetTests/input/api/yamlRenderer2.yml.pkl

```diff
@@ -18,6 +18,7 @@ res13 = new Dynamic { name = "pigeon"; age = 30 }
 res14 = new Person { name = "pigeon" }
 res15 = null
 res16 = Pair(1, 2)
+res17 = Bytes(1, 2, 3)
```

Comment:
Add tests for empty bytes, and, assuming we use RFC 2045 style encoding, bytes that exceed that length